From 19422cdac7618e63e665329542e074c6f27e7ba4 Mon Sep 17 00:00:00 2001
From: Alistair Francis <alistair.francis@wdc.com>
Date: Fri, 13 Jul 2018 15:14:58 -0700
Subject: [PATCH] linux-ptrace: Fix RISC-V cross compilation error

To avoid this error:
./gdb/nat/linux-ptrace.h:175:21: error: expected identifier      before '(' token
      #define TRAP_HWBKPT (4)
when cross compiling remove the #define.

Signed-eff-by: Alistair Francis <alistair.francis@wdc.com>
Upstream-status: Pending
---
 gdb/nat/linux-ptrace.h | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/gdb/nat/linux-ptrace.h b/gdb/nat/linux-ptrace.h
index 59549452c0..7f102a31ec 100644
--- a/gdb/nat/linux-ptrace.h
+++ b/gdb/nat/linux-ptrace.h
@@ -171,10 +171,6 @@ struct buffer;
 # define GDB_ARCH_IS_TRAP_HWBKPT(X) ((X) == TRAP_HWBKPT)
 #endif
 
-#ifndef TRAP_HWBKPT
-# define TRAP_HWBKPT 4
-#endif
-
 extern void linux_ptrace_attach_fail_reason (pid_t pid, struct buffer *buffer);
 
 /* Find all possible reasons we could have failed to attach to PTID
-- 
2.17.1

